src/main/scala/com/box/castle/core/leader/TaskManager.scala

```diff
@@ -59,20 +66,25 @@ class TaskManager(val leaderConfig: LeaderConfig, val committerConfigs: Iterable
       tasks
   }
 
-  private def matchTopicsRegex(topicsRegex: Regex, kafkaTopic: KafkaTopic, committerId: String, committerIds: Set[String]) = {
-    kafkaTopic.name match {
-      case topicsRegex(_*) => committerIds + committerId
-      case _ => committerIds
-    }
+  protected[leader] def matchTopicsRegex(topicsRegex: Regex, kafkaTopic: KafkaTopic, committerId: String, committerIds: Set[String]) = {
+      kafkaTopic.name match {
+        case topicsRegex(_*) => if(matchCommitterTopicFilter(kafkaTopic,committerId)) committerIds + committerId else committerIds
```

Comment:
1. Space between if & open paren ("(")
2. Since we have 2 conditions, would it make sense to change this to a simple if-statement, rather than using case and then a nested If?  (See matchTopicsSet(...) -- It seems the logic is quite similar, except the topicsSet(...) versus topicsRegex(...))